Package.swift

```diff
@@ -83,6 +83,6 @@ let package = Package(
                         .product(name: "SnapshotTesting", package: "swift-snapshot-testing")
                     ],
                     exclude: ["Templates/__Snapshots__", "Data/__Snapshots__", "TestPlans"],
-                    resources: [.copy("Resources/header.jpg"), .copy("Resources/background.jpg")])
+                    resources: [.copy("Resources/header.heic"), .copy("Resources/background.heic")])
```

Comment:
I attempted to replace this `jpg` with a `heic` to see if it fixed some flaky tests